DataDonationCryptoObjC/DataDonationCryptoObjCTests/Resources/TestModel/encryptedDataOutput.txt

```diff
@@ -0,0 +1 @@
+AgABtm3qXRKHOe8uk+EFjYAPFt8uPMs8sxZ37e5Af8pq8NG+mvT9Wsaq+nZ+8VZCnUILbipsOGOO7V5coqpTk+Mf4ID37ZXssF8ddP6eytv8/P4/L3hnTgKcwAm0Loxnxb76VTU3Xa3bZGV2LXbQh82oyNtnwaFk9SS0M4uAMHKLYwuQVV1/0cfHPyVMjWr7BzCzA2e/LVa39CZ9Lsc+IZVd6t/Bq/2LaX5hnOKNtr4FNQGehM4uVrYmPNCWekeBmzOimHPhjLoiKNq0x/TlraFoQSYjOz4SCl2cA2MIHsTVLETISI/84a3iscnjRcf3G4Qysg6iQc2kKGApEMBdRz5lfv4xoPzTcRR4Z4qh3RTmxqkkAAAAAAAAALGAISDgzyyXELuoMgCrMDU8FezBLmHUtFI5VkNFcv4UduaQUw==
```

Comment:
I am more and more convinced, we should have a gradle task which does copying from the common resources. Not necessary in this PR, but at least right after it.

Reply:
agreed.